cypress/fixtures/appData.js

```diff
@@ -37,6 +37,45 @@ export const MOCK_USER1_APP_DATA = [
   },
 ];
 
+export const MOCK_USER1_APP_DATA_2 = [
```

Comment:
I don't find this name really explicative. Could you have something more precise? Like what do you test with this specific mock data, in what is it different from other fixtures?

Reply:
I modified the name, and add a description to specify why I had to add more data.